src/main.c

```diff
@@ -1343,14 +1351,28 @@ static void socket_reuseport(int fd)
 /**
  * Opens an INET or INET6 socket for accepting connections. When the protocol is UDP, SO_REUSEPORT is set if available.
  */
-static int open_listener(int domain, int type, int protocol, struct sockaddr *addr, socklen_t addrlen)
+static int open_listener(int domain, int type, int protocol, struct sockaddr *addr, socklen_t addrlen, int timestamping)
 {
     int fd;
 
     if ((fd = socket(domain, type, protocol)) == -1)
         goto Error;
     set_cloexec(fd);
 
+   {
+        int flag = 1;
```

Comment:
Should we move this into `#if defined(__linux__)` below?

Reply:
sure, thanks!